authorizations/authorization-ranger/src/test/java/org/apache/gravitino/authorization/ranger/integration/test/RangerBaseE2EIT.java

```diff
@@ -514,9 +514,8 @@ void testWriteOnlyTable() throws InterruptedException {
     // case 1: Succeed to insert data into the table
     sparkSession.sql(SQL_INSERT_TABLE);
 
-    // case 2: Fail to select data from the table
-    Assertions.assertThrows(
-        AccessControlException.class, () -> sparkSession.sql(SQL_SELECT_TABLE).collectAsList());
+    // case 2: Fail select data from the table
```

Comment:
Please change comment. `Fail select?`

Reply:
Sorry, I corrected. You can take a look again.